Resources/Prototypes/Voice/speech_emotes.yml

```diff
@@ -247,6 +247,21 @@
   category: Vocal
   chatMessages: [Wehs!]
 
+- type: emote
+  id: WehChat
+  category: Vocal
+  chatMessages: [Weh!]
+  chatTriggers:
+    - weh
+    - weh.
+    - weh!
+    - wehs
+    - wehs.
+    - wehs!
+    - wehing
+    - wehing.
+    - wehing!
+
 - type: emote
```

Comment:
Isn't this an existing emote? Not sure why it needs 2 entirely different emotes.